cmd/task-runner/task-runner.go

```diff
@@ -143,6 +144,9 @@ func Run(build BuildFlags) {
 	cli := parseFlags(build)
 	glog.Infof("Task runner starting... version=%q", build.Version)
 
+	// Fire a metric to track which version of task-runner we're running
+	metrics.Version.WithLabelValues("task-runner", build.Version).Inc()
```

Comment:
@victorges open to moving this out of `Run()`, it was just less obvious where to put it within the `metrics` package

Reply:
Oh I thought there was a "high level" initialization function on the package, but maybe not in this project as the metric is also declared statically. If there's no such thing as a `metrics.Init` or `metrics.NewX`, this looks good

Reply:
Yeah, we don't have any metrics initialisation function at the moment, so will leave it here for now